README.md

```diff
@@ -3,7 +3,6 @@
 A GAN toolbox for researchers and developers with:
 - Progressive Growing of GAN(PGAN): https://arxiv.org/pdf/1710.10196.pdf
 - DCGAN: https://arxiv.org/pdf/1511.06434.pdf
-- To come: StyleGAN https://arxiv.org/abs/1812.04948
```

Comment:
:(

Reply:
So no StyleGAN implementation in the end, @Molugan ?😞

Reply:
There is a PR for styleGAN !

Reply:
styleGAN incoming (https://github.com/facebookresearch/pytorch_GAN_zoo/pull/95) so no need to remove this one